vmdb/app/controllers/ems_common.rb

```diff
@@ -265,6 +265,14 @@ def form_field_changed
           page << "miqValidateButtons('hide', 'amqp_');"
         end
       end
+      if @edit[:keypair_verify_status] != @edit[:saved_keypair_verify_status]
+        @edit[:saved_keypair_verify_status] = @edit[:keypair_verify_status]
+        if @edit[:keypair_verify_status]
+          page << "miqValidateButtons('show', 'keypair_');"
+        else
+          page << "miqValidateButtons('hide', 'keypair_');"
+        end
+      end
```

Comment:
ah, it needs to be

page << "miqValidateButtons('#{@edit[:keypair_verify_status] ? 'show' : 'hide'}', 'keypair_');"

the result needs to be:
miqValidateButtons('show', 'amqp_');
not:
miqValidateButtons(show, 'amqp_');